src/hooks/useWindowFocusEvent/index.website.ts

```diff
@@ -0,0 +1,3 @@
+import useWindowFocusEvent from './useWindowFocusEvent';
+
+export default useWindowFocusEvent;
```

Comment:
We can simply move the code inside `useWindowFocusEvent` to `index.ts` and then export the empty function for native from `index.native.ts`, it will make it a bit easier to navigate.

Reply:
Yes I've seen that in a few places and IMO it's bad practice 😄, but there's nothing in our style guide restricting that so I'll just update this to match 👍